File: app/src/main/java/com/jcs/where/features/address/AddressAdapter.java
package com.jcs.where.features.address;

import android.widget.ImageView;
import android.widget.TextView;

import com.chad.library.adapter.base.BaseQuickAdapter;
import com.chad.library.adapter.base.viewholder.BaseViewHolder;
import com.jcs.where.R;
import com.jcs.where.api.response.address.AddressResponse;

import org.jetbrains.annotations.NotNull;

/**
 * Created by Wangsw  2021/3/22 13:47.
 */
public class AddressAdapter extends BaseQuickAdapter<AddressResponse, BaseViewHolder> {

    TextView addressTv;
    TextView nameTv;
    TextView phoneTv;
    ImageView editIv;


    public AddressAdapter() {
        super(R.layout.item_address);
    }

    @Override
    protected void convert(@NotNull BaseViewHolder holder, AddressResponse data) {

        addressTv = holder.getView(R.id.address_tv);
        nameTv = holder.getView(R.id.name_tv);
        phoneTv = holder.getView(R.id.phone_tv);
        editIv = holder.getView(R.id.edit_iv);

        addressTv.setText(data.address);
        nameTv.setText(data.contact_name);
        phoneTv.setText(data.contact_number);



    }



}

File: app/src/main/java/com/jcs/where/features/address/AddressAdapter.kt
package com.jcs.where.features.address

import android.widget.ImageView
import com.chad.library.adapter.base.BaseQuickAdapter
import com.jcs.where.api.response.address.AddressResponse
import com.chad.library.adapter.base.viewholder.BaseViewHolder
import com.jcs.where.R
import android.widget.TextView

/**
 * Created by Wangsw  2021/11/4 17:22.
 *
 *
 */
class AddressAdapter : BaseQuickAdapter<AddressResponse, BaseViewHolder>(R.layout.item_address) {

    override fun convert(holder: BaseViewHolder, data: AddressResponse) {
        val addressTv = holder.getView<TextView>(R.id.address_name_tv)
        val nameTv = holder.getView<TextView>(R.id.name_tv)
        val  phoneTv = holder.getView<TextView>(R.id.phone_tv)
        var  editIv = holder.getView<TextView>(R.id.edit_iv)
        addressTv.text = data.address
        nameTv.text = data.contact_name
        phoneTv.text = data.contact_number
    }
}